<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_AddPictures</class>
 <widget class="QDialog" name="Dialog_AddPictures">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_browseInputDir">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>输入格式: f;0;ppx;0;f;ppy;0;0;1</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_inputDir"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_databaseDir"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_eigenMatrix"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_browseDatabaseDir">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>122</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>源图片参数设置：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>图片文件夹：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>相机数据库文件：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>相机本征矩阵：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_CONFIRM">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_CANCEL">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
